kupong_integration.gemspec

```diff
@@ -23,6 +23,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.14"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
```

Comment:
I put all the related API tests into MyWing. I have tests ready stubbed out API calls. But because i got some correct credential very late today, I wasn't able to finish them. Will create another PR for it... this has to go live tomorrow morning 🙄.